lib/sensors/include/FSR.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "mbed.h"
```

Comment:
I remember something about not needing to have mbed.h headers in out files. @younesr1  can you explain more
?